### DISCUSSION TITLE
Try to migrate from Minio to Seawedfs.

### DISCUSSION THREAD

**Discussion by meotimdihia** [seaweedfs/seaweedfs#2368] [Q&A]
I am using Minio to store images from 250kb -> 2MB. 
Currently with 10+ million images on a server with 16 cores,  storage is HDD.
And Minio always used 50-90% CPU iowait when transfer 200Mb/s
I don't know if Seaweedfs can help to solve this problem.

And my main question is: "MinIO strongly recommends local JBOD arrays for best performance. RAID or similar technologies do not provide additional resilience or availability benefits when used with distributed MinIO deployments, and typically reduce system performance."
Then what is Seaweedfs recommend?

**Comment by chrislusf** [comment]
Hard to tell what is the bottleneck for Minio in your case. Please let us know the perf difference when you finished migration.

JBOD should be fine.
